src/main/kotlin/no/nav/syfo/aktivitetskrav/AktivitetskravService.kt

```diff
@@ -154,10 +157,19 @@ class AktivitetskravService(
     suspend fun sendForhandsvarsel(
         aktivitetskrav: Aktivitetskrav,
         veilederIdent: String,
+        personIdent: PersonIdent,
         forhandsvarselDTO: ForhandsvarselDTO,
         callId: String,
     ): AktivitetskravVarsel {
-        val pdf = pdfGenClient.createForhandsvarselPdf(callId, forhandsvarselDTO.document)
+        val personNavn = pdlClient.navn(personIdent)
```

Comment:
Dette kallet øker vel responstiden litt for å sende forhåndsvarsel 🤔 Vi kan jo se på om det blir et problem og om hvorvidt vi skal ta inn Redis for å cache dette etterhvert 👍🏼

Reply:
Har så vidt begynt på en cache allerede, siden vi også henter navn ved journalføring.